backend/src/org/commcare/xml/StackOpParser.java

```diff
@@ -17,38 +14,34 @@
 /**
  * @author ctsims
  */
-public class StackOpParser extends ElementParser<StackOperation> {
```

Comment:
Gotcha. I know this is nitpicky, but given that we use the convention of putting `public` before our class names and constructors uniformly throughout the entire codebase, I think I'd rather not start changing it in random places just because the linter says so. It gives (at least me) the impression that it was done for a particular reason, since it's such a departure from our standard. If we want to make a separate pass at converting everything that can be to to package local, we can do that.